tests/components/alarm_control_panel/test_mqtt.py

```diff
@@ -6,12 +6,12 @@
     STATE_ALARM_DISARMED, STATE_ALARM_ARMED_HOME, STATE_ALARM_ARMED_AWAY,
     STATE_ALARM_PENDING, STATE_ALARM_TRIGGERED, STATE_UNAVAILABLE,
     STATE_UNKNOWN)
-from homeassistant.components import alarm_control_panel
+from homeassistant.components import alarm_control_panel, mqtt
 from homeassistant.components.mqtt.discovery import async_start
 
 from tests.common import (
     mock_mqtt_component, async_fire_mqtt_message, fire_mqtt_message,
-    get_test_home_assistant, assert_setup_component)
+    get_test_home_assistant, assert_setup_component, MockConfigEntry)
```

Comment:
This is a red flag. Your code does not impact alarm control panel but you still need to update the tests. Means you broke something 😉